app/services/deploy_event_service.rb

```diff
@@ -48,6 +48,14 @@ def service_already_exists!
     save_event("A service has been reused as it already existed")
   end
 
+  def unexpected_error!(error)
+    if error.is_a?(RestClient::ExceptionWithResponse)
+      save_event("#{error.class}: #{error}, Body: #{error.response}")
```

Comment:
These are stored in the DB right? Is there any limit on the length of the messages?